src/types/swiper-class.d.ts

```diff
@@ -304,10 +316,14 @@ interface Swiper extends SwiperClass<SwiperEvents> {
    */
   attachEvents(): void;
 
+  loopCreate(): void;
```

Comment:
Let's add `!INTERNAL` flag to loopedSlides, loopCreate and loopDestroy props. Don't want them to appear in docs